rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -394,7 +394,7 @@ const std::array<std::pair<ppu_intrp_func_t, std::string_view>, 1024> g_ppu_sysc
 	BIND_SYSC(sys_memory_container_destroy),                //342 (0x156)
 	BIND_SYSC(sys_memory_container_get_size),               //343 (0x157)
 	NULL_FUNC(sys_memory_budget_set),                       //344 (0x158)
-	null_func,//BIND_SYSC(sys_memory_...),                  //345 (0x159)
```

Comment:
Missing PPUFunction naming I suppose

Reply:
What do you mean?

Reply:
The function in PPUFunction.cpp has been removed a long time ago in favour of inlined naming in the this array.

Reply:
That's also fine because all syscall_0-1023 names are linked by default.

Reply:
https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/Cell/PPUThread.cpp#L3368